tailwind.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-  content: ["./src/**/*.{js,ts,tsx,mdx,stories.tsx}"],
+  content: ["./src/**/*.{js,ts,tsx,mdx,stories.tsx,json}"],
```

Comment:
Neat, so tailwind will scan this json file looking for class names?

Reply:
Yep, it's all a little bit arcane but it needs to have visibility over these generated JSON files